de.metas.endcustomer.mf15/de.metas.endcustomer.mf15.swingui/pom.xml

```diff
@@ -120,7 +120,27 @@
 			<version>${metasfresh.version}</version>
 			<scope>provided</scope>
 		</dependency>
-
+		
```

Comment:
> Maybe that's not the effort. It is the lack of experience on my side how to do this.

Right, i mean the effot of learning how to provide a PR without it :-)

> Feel free to exclude it from this pull-request.

I think i can't. AFAIK i can only merge a pull request as-is or not merge it.